Vagrantfile

```diff
@@ -0,0 +1,106 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
```

Comment:
Minor, but I'd recommend clearing out the comments in this file - they make it slightly harder for someone else to parse, and we can always recover them from our source control or the Vagrant documentation